### PR TITLE
Output logs when failures happen but not in a test

### PR DESCRIPTION
**Pull request by tateexon**
There is a corner case in go test results where the test run can fail but there are no test specified failures in the logs. We don't have a great way to pull out what caused it so in this case we just print out all the output.

### CODE REVIEW

File: go/go-test-results-parsing/src/main.ts [hunk 1]
```diff
@@ -49,16 +61,21 @@ export function deserializeTestResultsFile(resultsFile: string): TestResult[] {
...
+export function getTestFailures(testResults: TestResult[]): TestRunFailures {
   const names: string[] = []
+  let packageFailure = false
   testResults.filter(testResult => {
     if (testResult.Action === 'fail') {
       if (testResult.Test) {
         names.push(testResult.Test)
+      } else {
+        // we got a failure without a test failure
+        packageFailure = true
```
[the remark this answers is not in the document]
Comment (tateexon) [reply go/go-test-results-parsing/src/main.ts:73]: I am handling this above in the main function https://github.com/smartcontractkit/chainlink-github-actions/pull/104/files#diff-c7f06bc3c11b303038bdf62606b11c5bbbb495340e36f01702aaf4b39cebe347R29-R38

**Review by cedric-cordenier — APPROVED** [review]

File: go/go-test-results-parsing/src/main.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tateexon) [reply go/go-test-results-parsing/src/main.ts:75]: The comment in code has been updated to make more sense.

**Review by kalverra — APPROVED** [review]
